core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -21,6 +21,8 @@ import java.nio.ByteBuffer
 import java.util.{TimerTask, Timer}
 import java.util.concurrent.atomic.AtomicLong
 
+import org.apache.spark.scheduler.TaskLocality.TaskLocality
```

Comment:
This should be grouped with the other Spark imports, but I can probably fix this myself on merge.